index.html

```diff
@@ -188,7 +188,7 @@ <h2>
         playback device</a>. In such a case, both the <a>browsing context</a>
         and the media player run on the same UA and the operating system is
         used to route the player output to the <a>remote playback device</a>.
-        This is commonly referred to as the <dfn><b id="media-mirroring">media
+        This is commonly referred to as the <dfn><b>media
         mirroring</b></dfn> case. This specification imposes no requirements on
```

Comment:
Definitions already appear as bold terms. While we're at it, I'm wondering whether we can also drop the extra bold:

```suggestion
        This is commonly referred to as the <dfn>media
        mirroring</dfn> case. This specification imposes no requirements on
```